### PR TITLE
[CLI-2476] Add `confluent flink statement stop`

### PR DESCRIPTION
**Pull request by brianstrauch**
Release Notes
-------------
Bug Fixes
- Correctly format timestamps in YAML output

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----
Add stop command.

Test & Review
-------------
Integration tests

### CODE REVIEW

File: internal/flink/command_statement_stop.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	if err != nil {
+		return err
+	}
+
+	client, err := c.GetFlinkGatewayClient()
+	if err != nil {
+		return err
+	}
+
+	statement, err := client.GetStatement(environmentId, args[0], c.Context.LastOrgId)
```
Comment (sgagniere) [inline internal/flink/command_statement_stop.go:41]: Nitpick: `c.Context.GetCurrentOrganization()` returns `LastOrgId` w/ a nil check, so might be worth using instead.

**Review by sgagniere — APPROVED** [review]
+1

File: test/flink_test.go [hunk 2]
```diff
@@ -62,9 +62,10 @@ func (s *CLITestSuite) TestFlinkRegion() {
 func (s *CLITestSuite) TestFlinkStatement() {
 	tests := []CLITest{
 		{args: "flink statement delete my-statement --force --region eu-west-1 --cloud aws", fixture: "flink/statement/delete.golden"},
-		{args: "flink statement list --compute-pool lfcp-123456", fixture: "flink/statement/list.golden"},
 		{args: "flink statement describe my-statement --region eu-west-1 --cloud aws", fixture: "flink/statement/describe.golden"},
 		{args: "flink statement exception list my-statement --region eu-west-1 --cloud aws", fixture: "flink/statement/exception/list.golden"},
+		{args: "flink statement list --compute-pool lfcp-123456", fixture: "flink/statement/list.golden"},
```
Comment (MuweiHe) [inline test/flink_test.go:67]: test different output format?

File: test/fixtures/output/local/services/help-onprem.golden [hunk 3]
```diff
@@ -5,6 +5,7 @@ Usage:
 
 Available Commands:
   connect         Manage Connect.
+  control-center  Manage Control Center.
```
Comment (brianstrauch) [inline test/fixtures/output/local/services/help-onprem.golden:8]:
```suggestion
```

File: test/fixtures/output/local/services/help.golden [hunk 4]
```diff
@@ -5,6 +5,7 @@ Usage:
 
 Available Commands:
   connect         Manage Connect.
+  control-center  Manage Control Center.
```
Comment (brianstrauch) [inline test/fixtures/output/local/services/help.golden:8]:
```suggestion
```

**Pull request merged**